tensorflow/core/platform/default/build_config.bzl

```diff
@@ -670,8 +670,7 @@ def tf_additional_core_deps():
         clean_dep("//tensorflow:android"): [],
         clean_dep("//tensorflow:ios"): [],
         clean_dep("//tensorflow:linux_s390x"): [],
-        clean_dep("//tensorflow:windows"): [],
-        clean_dep("//tensorflow:with_tpu_support"): [],
+        clean_dep("//tensorflow:no_hdfs_support"): [],
```

Comment:
I think we should keep the `windows` label here?

Reply:
Maybe it's better to use multiple `select`s if these are removed because of confusing labels?

Reply:
@mihaimaruseac The `windows` and `with_tpu_support` was removed was to match the similar settings above in gcs (line 661-668):
```
select({
        clean_dep("//tensorflow:android"): [],
        clean_dep("//tensorflow:ios"): [],
        clean_dep("//tensorflow:linux_s390x"): [],
        clean_dep("//tensorflow:no_gcp_support"): [],
        "//conditions:default": [
            "//tensorflow/core/platform/cloud:gcs_file_system",
        ],
    }) 
```

and s3 settings below (line 677-line 685):
```
    }) + select({
        clean_dep("//tensorflow:android"): [],
        clean_dep("//tensorflow:ios"): [],
        clean_dep("//tensorflow:linux_s390x"): [],
        clean_dep("//tensorflow:no_aws_support"): [],
        "//conditions:default": [
            clean_dep("//tensorflow/core/platform/s3:s3_file_system"),
        ],
    })
```

I actually think maybe we can just simplify and use 2-way select of:
```
select({
        clean_dep("//tensorflow:no_hdfs_support"): [],
        "//conditions:default": [
            clean_dep("//tensorflow/core/platform/hadoop:hadoop_file_system"),
        ],
    }) 
```

Maybe I can give it a try?